test_scripts/Defects/4_5/1211_ATF_Check_count_of_removals_for_bad_behavior_REQUEST_WHILE_IN_NONE_HMI_LEVEL.lua

```diff
@@ -0,0 +1,79 @@
+----------------------------------------------------------------------------------------------------
+-- Script verifies issue https://github.com/SmartDeviceLink/sdl_core/issues/1211
+-- Flow: EXTERNAL_PROPRIETARY
+----------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require("user_modules/script_runner")
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local mobile_session = require("mobile_session")
+local common = require("test_scripts/Defects/4_5/commonDefects")
+local commonTestCases = require("user_modules/shared_testcases/commonTestCases")
+
+--[[ General configuration parameters ]]
+config.ExitOnCrash = false
+
+-- [[Local variables]]
+local count_of_requests = 10
+
+local function updateINIFile()
+  common.backupINIFile()
+  commonFunctions:write_parameter_to_smart_device_link_ini("AppHMILevelNoneTimeScaleMaxRequests", count_of_requests)
+  commonFunctions:write_parameter_to_smart_device_link_ini("AppHMILevelNoneRequestsTimeScale", 30000)
+end
+
+local function registerApp(self)
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession:StartService(7)
+  :Do(function ()
+      local cid = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+      EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered")
+      EXPECT_RESPONSE(cid, { success = true })
+      EXPECT_NOTIFICATION("OnPermissionsChange")
+    end)
```

Comment:
Add expectation of OnHMIStatus(NONE) to be sure that app is set in NONE level.

Reply:
@GetmanetsIrina Done in 440b713